app/src/main/java/com/github/fribourgsdp/radio/LobbyActivity.kt

```diff
@@ -210,17 +235,33 @@ open class LobbyActivity : AppCompatActivity() {
         gameBuilder.setUserList(playersList.map { name -> User(name) })
     }
 
-    private fun goToGameActivity(isHost: Boolean, game: Game? = null, gameID: Long? = null) {
+    private fun goToGameActivity(isHost: Boolean, game: Game? = null, gameID: Long) {
         val intent: Intent = Intent(this, GameActivity::class.java).apply {
             putExtra(GAME_IS_HOST_KEY, isHost)
+            putExtra(IS_IN_TEST_MODE, false)
```

Comment:
Same as above comment?

Reply:
Yes indeed.

Reply:
No it's an oversight thank you for pointing it out.